config/webpack/webpack.config.js

```diff
@@ -73,7 +76,7 @@ const makeWebpackConfig = ({ isIntegration = false, port = 0 } = {}) => {
 
   const createEntry = entry => [
     ...resolvedPolyfills,
-    ...(isStartScript ? devServerEntries : []),
+    ...(hot ? devServerEntries : []),
```

Comment:
🤔 Not sure we should use `hot` instead of `isStartScript` in some of these scenarios. `sku start` currently doesn't support hot reloading right?

Reply:
Yeah happy to bikeshed this. `isStartScript` is suffering a couple of problems at the minute:
1. It's not triggered when using the `test-manual` script,
2. Should be true outside of `start`, eg. `storybook`, `playroom` etc.